packages/react-scripts/template/README.md

```diff
@@ -1327,6 +1294,39 @@ The test command will force Jest to run tests once instead of launching the watc
 
 The build command will check for linter warnings and fail if any are found.
 
+### Pre-commit Hook
+
+You can run tests against "staged" files before each Git commit by integrating the test script in pre-commit hook.
```

Comment:
Is it really needed to add docs about prettier in CRA? since there's no special step to do it in a CRA based project.
people can read through `prettier` docs then do exactly what the docs say, it'll just work. and IMO by merging #2006 will make CRA docs bloated day by day.

on the other hand, i think we should add docs & explain something which are specific/special in CRA based project like how to run test, storybook, coverage reporting, linting, etc. what do you think?